.gitignore

```diff
@@ -1,4 +1,3 @@
-.vscode
```

Comment:
The settings.json and c_cpp_properties.json files are already being tracked, so having this in the .gitignore does nothing. We should probably remove these and .gitignore to allow people overriding code-workspace settings, or we should start using multiple code-workspace files.